obd/OBDResponse.py

```diff
@@ -135,14 +140,13 @@ def __len__(self):
     def __getitem__(self, key):
         if isinstance(key, int):
             return self._tests.get(key, MonitorTest())
-        elif isinstance(key, str) or isinstance(key, unicode):
```

Comment:
To make it working without syntax error in python3, you need to create `string_types` variable depending on current python version - as here at the top of the file (because there is no `unicode` keyword in python3).